omod/src/main/java/org/openmrs/module/fhir/providers/RestfulObservationResourceProvider.java

```diff
@@ -120,4 +122,17 @@ public Observation getResourceById(@IdParam IdDt theId) {
 			                                                 answerConceptName) {
 		return provider.searchObsByValueConcept(answerConceptName);
 	}
+
+	/**
+	 * Search observations by patient identifier
+	 *
+	 * @param identifier object containing the patient identifier
+	 */
+	@Search()
+	public List<Observation> searchEncountersByPatientIdentifier(
+		@RequiredParam(name=Observation.SP_PATIENT, chainWhitelist= {Patient.SP_IDENTIFIER}) ReferenceParam identifier
```

Comment:
As I mentioned earlier formatting and whtz d need of chainwhitelist? how about following sysntax?
@RequiredParam(name=Patient.SP_IDENTIFIER) ReferenceParam identifier